.docker-compose-db.yaml

```diff
@@ -9,8 +9,8 @@ services:
     restart: always
     ports:
     - "5432:5432"
-    environment:
```

Comment:
These dbs should not be exposed externally, so pretty pointless to set up authentication here.